fire/cli/info.py

```diff
@@ -467,13 +526,35 @@ def punkt(ident: str, obs: str, koord: str, **kwargs) -> None:
         'sagseventtilid',     -- sagsevent for punktinvalideringen\n
         'slettet'             -- øh\n
     )
+
+    Anfører man ikke specifikke tilvalg vises kun basale dele: Attributter og
+    punktbeskrivelser + gældende koordinater.
+
+    Tilvalg `--detaljer/-D` udvider med sjældnere brugte informationer
+
+    Tilvalg `--koord/-K` kan sættes til ts, alle, ingen - eller kombinationer:
+    fx ts,alle. `alle` tilvælger historiske koordinater, `ts` tilvælger
+    tidsseriekoordinater, `ingen`fravælger alle koordinatoplysninger.
+
+    Tilvalg `--obs/-O` kan sættes til alle eller niv. Begge tilvælger visning
+    af observationer til/fra det søgte punkt. P.t. understøttes kun visning af
+    nivellementsobservationer.
     """
     pi = aliased(PunktInformation)
     pit = aliased(PunktInformationType)
 
+    # Vær mindre pedantisk mht. foranstillede nuller hvis identen er et landsnummer
+    landsnummermønster = re.compile("[0-9]*-[0-9]*-[0-9]*$")
+    if landsnummermønster.match(ident):
+        dele = ident.split("-")
+        herred = int(dele[0])
+        sogn = int(dele[1])
+        lbnr = int(dele[2])
+        ident = f"{herred}-{sogn:02}-{lbnr:05}"
+
     # Forsøg at finde et bedre bud hvis identen er en UUID
-    uuidpattern = re.compile("[a-f0-9]*-[a-f0-9]*-[a-f0-9]*-[a-f0-9]*-[a-f0-9]*")
-    if uuidpattern.match(ident):
+    uuidmønster = re.compile("[a-f0-9]*-[a-f0-9]*-[a-f0-9]*-[a-f0-9]*-[a-f0-9]*$")
+    if uuidmønster.match(ident):
         ident = kanonisk_ident(ident)
```

Comment:
Er det her ikke at gå over åen efter vand? Først slår du et punkt op på dets UUID for at at finde et ident, som du derefter kan bruge til at slå det's UUID op. Hvis du får et UUID, så forsøg da bare at hive punktet ud på det. Hvis du så laver `kanonisk_ident()` om til en property på `Punkt` så har du stadig let adgang til et brugbart ident.

Reply:
Jo - det er en fejl: Jeg har haft det udkommenteret undervejs (der var lidt problemer med logikken ifht. `firedb.hent_punkt`, som kaldes ved uuid, vs. `firedb.session.query`, som er rettet for længst), så de linjer skulle slettes frem for af-udkommenteres. Kommer i et snarligt commit